ehrql/utils/log_utils.py

```diff
@@ -34,11 +16,10 @@ def init_logging():
             "disable_existing_loggers": False,
             "formatters": {
                 "formatter": {
-                    "()": structlog.stdlib.ProcessorFormatter,
-                    "processor": structlog.dev.ConsoleRenderer(
-                        pad_event=0, sort_keys=False
-                    ),
-                    "foreign_pre_chain": pre_chain,
+                    "()": EHRQLFormatter,
+                    "format": "{asctime} [{levelname_lower:<9}] {message}",
+                    "datefmt": "%Y-%m-%d %H:%M:%S",
```

Comment:
wot, no timezone?

Reply:
Also, I've been somewhat reluctant to use the Z prefix, as python doesn't parse it :(. And +NN:00 is a lot of noise.

```
>>> from datetime import datetime
>>> datetime.fromisoformat("2024-11-06 12:34:56Z")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: Invalid isoformat string: '2024-11-06 12:34:56Z'
>>> datetime.fromisoformat("2024-11-06 12:34:56+00:00")
datetime.datetime(2024, 11, 6, 12, 34, 56, tzinfo=datetime.timezone.utc)
```

Reply:
I know you've said you weren't trying to alter existing behaviour, but I wonder if now is the time to think about this stuff simply because it's going to involve a visual change for users in any case (dropping the colourisation) so it might make sense to do this all at once.

What do you think about removing the timestamps entirely? It's redundant noise in the production logs where docker adds them for us. And I'm not sure they have any use locally.

Reply:
I am for removing them. I don't think they add value for the user, probably just noise. And it sidesteps the UTC question too.